pkg/cmd/server/kubernetes/master/master_config.go

```diff
@@ -369,6 +375,60 @@ func BuildControllerManagerServer(masterConfig configapi.MasterConfig) (*cmapp.C
 	return cmserver, cloud, nil
 }
 
+// AddRecyclerTemplates creates a file with default recycled pod template and
+// adds it into PersistentVolumeRecyclerConfiguration if it is not set. It
+// returns name of the template file that should be deleted after controllers
+// start.
+func AddRecyclerTemplates(masterConfig configapi.MasterConfig, cmserver *cmapp.CMServer) (string, error) {
```

Comment:
Sort of. I tried it and it looked extremely ugly. If I create a temp file in BuildControllerManagerServer, I need to delete it in all error paths between `BuildControllerManagerServer` call and `startControllers`. I can't use `defer` because `startControllers` runs in a separate goroutine and `Master.Start` might actually finish earlier than `startControllers` is called.